go.mod

```diff
@@ -41,7 +40,7 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/ti-mo/conntrack v0.3.0
 	github.com/vishvananda/netlink v1.1.0
-	github.com/vmware/go-ipfix v0.4.7
+	github.com/vmware/go-ipfix v0.2.1-0.20210503231621-19760d7cef9d
```

Comment:
this doesn't seem right? you're reverting to an older version?

Reply:
This is temporary which is created based on commit hash..  will move to the right tag around the time of merging this PR.